Formula/netdata.rb

```diff
@@ -0,0 +1,62 @@
+class Netdata < Formula
+  desc "Distributed real-time performance and health monitoring"
+  homepage "https://my-netdata.io/"
+  url "https://github.com/firehol/netdata/releases/download/v1.9.0/netdata-1.9.0.tar.bz2"
+  sha256 "542b4799aed1ee03b0a0dbd00fae988a788622c431382e6429362fbbb4f0f017"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ossp-uuid"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}"
+    system "make", "install"
+
+    conf_path = (etc/"netdata")
+    conf_path.mkpath
+    conf_path.install "system/netdata.conf"
+  end
+
+  def post_install
+    inreplace (etc/"netdata/netdata.conf"), /web files owner = .*/, "web files owner = #{ENV["USER"]}"
+    inreplace (etc/"netdata/netdata.conf"), /web files group = .*/, "web files group = #{group}"
+  end
+
+  def group
+    Etc.getgrgid(prefix.stat.gid).name
+  end
+
+  plist_options :manual => ".../sbin/netdata -D"
```

Comment:
I'm not sure what to write here now when it's not on the path.

Reply:
How about,
```rb
plist_options :manual => "#{HOMEBREW_PREFIX}/sbin/netdata -D"
```

Reply:
Fixed